config/config.go

```diff
@@ -164,8 +162,6 @@ type Config struct {
 	EnableGlobalIndex bool `toml:"enable-global-index" json:"enable-global-index"`
 	// DeprecateIntegerDisplayWidth indicates whether deprecating the max display length for integer.
 	DeprecateIntegerDisplayWidth bool `toml:"deprecate-integer-display-length" json:"deprecate-integer-display-length"`
-	// EnableRedactLog indicates that whether redact log, 0 is disable. 1 is enable.
-	EnableRedactLog int32 `toml:"enable-redact-log" json:"enable-redact-log"`
```

Comment:
This will break the compatibility, should mention in the release note?

Reply:
s/EnableRedactLog indicates that whether redact log, 0 is disable. 1 is enable./EnableRedactLog indicates whether to redact log, 0 for disable, 1 for enable./

Just a quick glance for grammatical change.